Feature/erd for ruby

neoladsouza opened the pull request:
We created the documentation for the ERDiagram based on Ruby's schema.

Commit:
LIBITD-2529. Added Ruby version of ERDiagram

We created an ERDiagram based on the Ruby
schema for the RubyOnRails IPManager.

https://umd-dit.atlassian.net/browse/LIBITD-2529

Commit:
LIBITD-2529. Put RubyERDiagram in correct folder

Made a new folder called "docs".
Renamed RubyERDiagram.md to rails-model.md, and moved it to docs.

https://umd-dit.atlassian.net/browse/LIBITD-2529

peichman-umd reviewed (changes requested):
Looks accurate to me. One thing regarding ERD conventions: Entities should be named using singular nouns. See <https://mermaid.js.org/syntax/entityRelationshipDiagram.html>:

> an ER diagram shows [...] abstract instances, and the relationships between them. This is why entities are always named using singular nouns.

Please change the entity names to their singular form.

Commit:
LIBITD-2529. Changed plural names to singular in rails-model.md

Resolved comment and changed the names of the tables to be singular.

https://umd-dit.atlassian.net/browse/LIBITD-2529
